tfx/examples/imdb/README.md

```diff
@@ -31,8 +32,12 @@ if __name__ == '__main__':
   df['text'] = df['text'].str.decode("utf-8")
   dst_path = os.getcwd() + '/data/imdb.csv'
   df.to_csv(dst_path, index=False)
+
+# This code is available in imdb_fetch_data.py which you can run, or import
+# to another file.
 ```
```

Comment:
I think that we can move the entire code block and just leave a description here. For example,

* See [imdb_fetch_data.py](imdb_fetch_data.py) to use the entire imdb dataset.

We generally don't want to duplicate any code pieces.

Reply:
Got it.
What about the Dataset Source Acknowledgement, do I remove it from the README?

Reply:
I think that it should be in README because we have a fraction of the dataset in https://github.com/tensorflow/tfx/tree/master/tfx/examples/imdb/data .

Thanks!